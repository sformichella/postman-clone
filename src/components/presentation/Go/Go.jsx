import React from "react";

export default function Go() {
  return (
    <>

    </>
  );
}
