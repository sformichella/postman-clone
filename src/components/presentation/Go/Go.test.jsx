import React from "react";
import { render, cleanup } from "@testing-library/react";
import Go from "./Go";

describe("Go component", () => {
  afterEach(() => cleanup());
  it("renders Go", () => {
    const { asFragment } = render(<Go />);
    expect(asFragment()).toMatchSnapshot();
  });
});
