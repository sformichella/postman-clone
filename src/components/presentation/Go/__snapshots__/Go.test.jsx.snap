// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Go component renders Go 1`] = `
<DocumentFragment>
  <button>
    Query!
  </button>
</DocumentFragment>
`;
